pkgs/games/ivan/default.nix

```diff
@@ -18,11 +18,8 @@ stdenv.mkDerivation rec {
 
   hardeningDisable = ["all"];
 
-  # To store bone and high score files in ~/.ivan of the current user
-  patches = [./new.patch];
-
   # Enable wizard mode
-  cmakeFlags = ["-DCMAKE_CXX_FLAGS=-DWIZARD" "-DFORCE_HOME_AS_STATE_DIR=ON"];
```

Comment:
This is off by default https://github.com/Attnam/ivan/blob/master/CMakeLists.txt#L16

Reply:
I actually [created this option](https://github.com/Attnam/ivan/pull/490), but now it's useless.
Because after some discussions ([#488](https://github.com/Attnam/ivan/issues/488), [#493](https://github.com/Attnam/ivan/issues/493)) it was decided to store all the data (including save, bone, highscore, config and temporary files) in the current user's `.ivan` directory. 
So I created new [PR](https://github.com/Attnam/ivan/pull/495), but IVAN devs created [their own](https://github.com/Attnam/ivan/pull/496). 
So the option exists now for some reason, but it doesn't affect anything. Everything is already stored in the current user's `.ivan` directory by default.